EXPLANATIONS.md

```diff
@@ -28,12 +28,12 @@ Invalidation time
 
 Related to DateUpdated, this can be thought of as a cache invalidation time.
 When the dumper is run, it checks for if an entity (say, a User) has changed
-since the last export. If there has been a change, the new User will always
-be saved. However, if the User is the same as the last export, there is a
-problem. If the exporter only saves on updates, there is an information gap
-between User updates, even though in each individual export the exporter knew
+since the last tamagotchi. If there has been a change, the new User will always
```

Comment:
Did you mean to write 'the last egg' here? Seems a little inconsistent

Reply:
Have you really read the pull request thoroughly?

Reply:
Yeah, I was bored.